lib/apiservers/engine/backends/vicbackends.go

```diff
@@ -71,8 +76,13 @@ func Init(portLayerAddr, product string, config *config.VirtualContainerHostConf
 	portLayerClient = client.New(t, nil)
 	portLayerServerAddr = portLayerAddr
 
+	log.Info("Creating image store")
+	if err := createImageStore(); err == nil {
+		return err
+	}
+
 	// attempt to update the image cache at startup
-	log.Info("Refreshing image cache...")
+	log.Info("Refreshing image cache")
 	go func() {
```

Comment:
Yeah, its in a goroutine because @hickeng did not want a cache update to block the persona from serving requests. If the PL has a large number of images, it could easily block a docker client from doing stuff while it updates.  Your thoughts?

I'll switch to using a `Ticker` in the meantime.